python/private/attributes.bzl

```diff
@@ -282,6 +282,12 @@ that depend on this rule. The strings are repo-runfiles-root relative,
 
 Absolute paths (paths that start with `/`) and paths that references a path
 above the execution root are not allowed and will result in an error.
+
+:::{attention}
+Setting both this and the {attr}`site_packages_root` attribute may result in
+undefined behavior. Both will result in the code being importable, but from
```

Comment:
Should we error out here?

Reply:
Hm. Yeah. I can't imagine why one would want it on sys.path twice.

Even in the case where there are conflicts creating the symlinks, the later entry on sys.path from the imports attr wouldn't help -- sys.path is coming first. I suppose it would help for namespace packages but, if they're namespace packages, then the files probably wouldn't be conflicting?

Yeah, I'm having a hard time seeing the value in both being set. I'll change it to fail.

Reply:
Done